udata_front/theme/gouvfr/templates/organization/display.html

```diff
@@ -31,180 +43,322 @@
+        </div>
+    </div>
+    <div
+        id="reuses-panel"
+        class="fr-tabs__panel"
+        role="tabpanel"
+        aria-labelledby="reuses"
+        tabindex="0"
+    >
+        <div class="fr-container">
+            {% if reuses %}
+                <ul class="fr-grid-row fr-grid-row--gutters fr-mb-2w">
+                    {% for reuse in reuses %}
+                    <li class="fr-mb-3v fr-col-lg-3 fr-col-sm-6 fr-col-12">
+                        {% include theme('reuse/card.html') %}
+                    </li>
+                    {% endfor %}
+                </ul>
+                {{ paginator(reuses, arg_name='reuses_page', url='#/reuses') }}
+            {% else %}
+                <div class="fr-grid-row flex-direction-column fr-grid-row--middle fr-mt-5w">
+                    {% include theme('svg/blank_state/reuse.svg') %}
+                    <p class="fr-h6 fr-mt-2w fr-mb-5v text-center">
+                        {{ _("This organization hasn't published any reuses yet.") }}
+                    </p>
+                    <p>
+                        <a
+                            class="fr-btn fr-btn--secondary fr-btn--secondary-grey-500 fr-ml-1w"
+                            href="{{ config.GUIDES_REUSE_URL }}"
+                        >
+                            {{ _("What's a reuse ?") }}
+                        </a>
+                    </p>
                 </div>
-            </dl>
+            {% endif %}
         </div>
-        <div class="fr-col-md-6 fr-col-12 fr-my-3w fr-my-md-0">
-            <h3>{{ _('Actions') }}</h3>
-            <ul class="nav-list fr-ml-n3v">
-                {% if can_edit and not read_only_mode %}
-                <li class="fr-my-2w">
-                    <a
-                        class="nav-link nav-link--no-icon fr-raw-link text-decoration-none fr-link fr-link--icon-left fr-icon-arrow-right-s-line"
-                        href="{{ url_for('admin.index', path='organization/{id}/'.format(id=org.id)) }}"
-                    >
-                        <span class="text-decoration-underline">{{ _('Modify information') }}</span>
-                    </a>
-                </li>
-                {% endif %}
-                <li class="fr-my-2w">
-                    <a
-                        href="{{ url_for('organizations.datasets_csv', org=org) }}"
-                        class="nav-link nav-link--no-icon fr-raw-link text-decoration-none fr-link fr-link--icon-left fr-icon-arrow-right-s-line"
-                    >
-                        <span class="text-decoration-underline">{{ _('Download list of datasets as csv file') }}</span>
-                    </a>
-                </li>
-
+    </div>
+    <div
+        id="information-panel"
+        class="fr-tabs__panel"
+        role="tabpanel"
+        aria-labelledby="information"
+        tabindex="0"
+    >
+        <div class="fr-container">
+        {% call() information_section(_('Description')) %}
```

Comment:
We shouldn't have the description once again here? It already appears in `presentation` panel

Reply:
You're right ! I don't know what Figma I was looking at but it's not here anymore